### PR TITLE
Adds Fernet, a digestif that reduces your satiety.

### PR DESCRIPTION
**Pull request by ThePainkiller**
:cl: Dimmadunk
add: There's a new type of reagent added to the booze dispenser: Fernet. With it you can make a couple of new digestif drinks that will help you reduce excess fat!
/ :cl:

Have you ever bit more than you could chew and regretted it later?
Have you ever wanted to chew on a Gaia leaf to heal yourself but you can't force any more food down your throat?
Did you ever want to reduce your satiety FAST without poisoning you in the process with lipolicide?
Do you just like getting fucking smashed at the bar with a higher drink repertoire?

Introducing **Fernet**. A world-wide known digestif of italian origin commonly mixed with cola, with three recipes to be made out of it for different palates:

**Fernet Cola:** the classic, helps reducing your hunger

**Fanciulli:** the REAL way to prepare a Manhattan (since it ACTUALLY uses a bitter!), a sobering drink that makes you lazy (causes an initial tick of stamina damage the first time it enters your body)

**Branca Menta:** the refreshing one, perfect for a hot summer afternoon after eating too much meat. It cools you down if overheated, and makes you even more lazy than Fanciulli.

The marvelous spriting work is all done by [Dantebumer](http://www.byond.com/members/Dantebumer).

Values are more than open to change due to balance reasons as mantainers see fit. I do not want to make this drink look like a cheap tirizene.

### CODE REVIEW

**Comment by Wilchenx** [comment]
Remember to also update our wiki

**Comment by ThePainkiller** [comment]
If it gets approved and merged, I will update the wiki as well!

It does need more updating since it doesn't have grenadine, I had to dive on the code to see how a tequila sunrise was made.

**Comment by RicardoFort** [comment]
HACEME UN PR
HACEME UN PR
QUE ESTA NOCHE
QUIERO GOZAR

File: code/modules/reagents/chemistry/reagents/alcohol_reagents.dm [hunk 1]
```diff
@@ -1612,4 +1612,93 @@ All effects don't start immediately, but rather get worse over time; the rate is
...
+	return ..()
+
+/datum/reagent/consumable/ethanol/fanciulli
+
+	name = "Fanciulli"
+	id = "fanciulli"
+	description = "What if the Manhattan coctail ACTUALLY used a bitter herb liquour? Helps you sobers up." //also causes a bit of stamina damage to symbolize the afterdrink lazyness
+	color = "#CA933F" // rgb: 202, 147, 63
+	boozepwr = -10
+	taste_description = "a sweet sobering mix"
```
Comment (Barhandar) [inline code/modules/reagents/chemistry/reagents/alcohol_reagents.dm]: Does taste system not handle the a/an particle by itself?

**Comment by ExcessiveUseOfCobblestone** [comment]
Why is furnetpuro a sprite? looks like the normal drinking glass?

**Comment by ThePainkiller** [comment]
Let's play the _"how many layers of sleep deprivation are you on while coding"_ game
Excuse me for being this dumb and keep correcting my own messes

**Comment by ThePainkiller** [comment]
@ExcessiveUseOfCobblestone It's a drinking glass filled with a certain gradient of syrupy, very dark green
We wanted it to NOT look like the Irish Car Bomb, but still make it look different than a drinking glass filled with Charcoal.

**Comment by ExcessiveUseOfCobblestone** [comment]
@ThePainkiller I haven't looked at these sprites in 5ever since I deleted half of them for the exact same reason but if you can do it through reagent color please do that over new sprite.

**Comment by ThePainkiller** [comment]
@ExcessiveUseOfCobblestone that would require to just remove the glass_icon_state line, wouldn't it? It would become just a reagent inside a glass with its hex value showing as liquid.

**Comment by ExcessiveUseOfCobblestone** [comment]
Yep! I tried to make it simple when I coded that sucker 200000 years ago

**Comment by ExcessiveUseOfCobblestone** [comment]
Also if you plan on making more cool features please hop on #coderbus (irc) or Coding-General (discord).

I see you are making edits on master and it becomes a real pain real fast. Don't want you to get discouraged because of the funs of git lol.

**Comment by ThePainkiller** [comment]
Sorry, I'm still trying to get the gist of github and PR's, other than fixing grammar mistakes here and there, but then again I blame my sleep deprivation and fatigue
This is truly my first huge pr editing multiple files (and actually adding content)

**Comment by Tlaltecuhtli** [comment]
use the web editor its gr8
when looking at a file u press edit (make sure u on master branch, switch it in the top left of the file)
change the line and then merge it into thr branch used for this pr

**Comment by Iamgoofball** [comment]
This is cool but we do already have a weight loss reagent. Lipolicide.

**Comment by PKPenguin321** [comment]
That is not how you make a changelog, I don't think that will actually work at all

**Comment by davethwave** [comment]
> This is cool but we do already have a weight loss reagent. Lipolicide.

Lipolicide is a toxin that deals toxin damage and probably more effective and dangerous, while this is made for casual use and doesn't sap your hunger as much per unit.

**Comment by juanototo2** [comment]
image

**Comment by ExcessiveUseOfCobblestone** [comment]
This also shifts the downside to potential liver damage which is a lot more longterm than just toxin.

**Comment by ThePainkiller** [comment]
@ExcessiveUseOfCobblestone The thing is that it's supposed to not be binge-drank, just take a bit, reduce your satiety, wait out the stamina loss and be on your way. I'd say it's a decent trade off for lipolicide being harder to get (either by hacking botany and actually bothering to plant cannabis, or combining diethylamine ephedrine and whatever else made it)
On a side note, one contains negative boozepwr (since it's a sobering coctail) and the fernet cola has the same boozepwr value as beer (25), wich is on the second lowest tier of alcoholic power.
As I said, I'm always open for reccomendations on how to balance it out to make it coherent!

@PKPenguin321 I apologize, I am still learning the ropes on how to format my PR's and I was told to do the cl slash cl thing, but they never bothered explaining me how the :cl: works, or why the fuck do I need to format it like that. I will fix it to better format it to be changelog compliant. Something like:

:cl: Dimmadunk
add: There's a new type of reagent added to the booze dispenser: Fernet. With it you can make a couple of new digestif drinks that will help you reduce excess fat!
/ :cl:

Will that work?

**Comment by Barhandar** [comment]
Yes, but formatting and its workings are obvious from how it's formatted in the automatic example on opening the PR. Unless you did that somehow bypassing the text.

**Comment by ThePainkiller** [comment]
Time to edit the wiki it seems! <3

**Comment by Aranclanos** [comment]
based

**Pull request merged**